pnpm-lock.yaml

```diff
@@ -99,18 +99,6 @@ importers:
       rollup: 2.67.3
       typescript: 4.1.2
 
-  packages/auto-install/test/fixtures/pnpm:
-    specifiers:
-      node-noop: ^1.0.0
-    dependencies:
-      node-noop: 1.0.0
-
-  packages/auto-install/test/fixtures/pnpm-bare:
-    specifiers:
-      node-noop: ^1.0.0
-    dependencies:
-      node-noop: 1.0.0
-
```

Comment:
I have no idea what this is about.  `pnpm install` removed these lines.  I'm happy to manually revert.

Reply:
as long she builds, we're good